utils/scripts/tests/test_import_achievement_resources.py

```
@@ -0,0 +1,2458 @@
+            ),
+            mock.patch.object(
+                IMPORTER, "_load_components", return_value=()
+            ),
+            mock.patch.object(
+                IMPORTER, "_load_component_counts", return_value=()
+            ),
+        ):
+            with self.assertRaisesRegex(
+                IMPORTER.ResourceError,
+                "AchievementCategories.txt: expected at least one data row",
+            ):
```

Ruff (0.16.1)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)